### ISSUE TITLE
Android App: Unable to Upload Photos or Videos in Chat; Web Version Has Extremely Long Load Times


### ISSUE THREAD

**Issue by MRJOHN5ON** [RocketChat/Rocket.Chat#34842]
# **Description**:

In the Rocket.Chat Android app, clicking the "+" button next to the message input field only provides the option to "Create Discussion." There is no option to upload photos or videos directly within the app. Users are forced to switch to the web browser version, which provides the expected options (as seen in the attached screenshot). However, the web version takes an excessively long time to load, sometimes 2-3 minutes on both desktop and mobile devices, making this workaround impractical.

# **Steps to reproduce:**

1. Open the Rocket.Chat Android app.

2. Navigate to any chat.

3. Tap the "+" button next to the message input field.

4. Observe that the only option available is "Create Discussion."

5. Switch to the web version and note the additional media upload options that are missing in the app.

# **Expected behavior:**

The "+" button in the Android app should display the same options available in the web version, including the ability to upload photos and videos.

# **Actual behavior:**

The "+" button in the Android app only provides the "Create Discussion" option. Media uploads are not supported, requiring users to rely on the web version, which has extremely slow load times.

**# Screenshots:**

**1. Android App Interface**

Description: This shows the limited options in the Android app where only "Create Discussion" is available.
![Screenshot_20241228_104459_RocketChat.jpg](https://github.com/user-attachments/assets/8a48e4b5-314b-4909-a0b4-cfa519501177)

**2. Web Browser Interface**

Description: This shows the expected media upload options available in the web browser version.
![Screenshot_20241228_104320_Chrome.jpg](https://github.com/user-attachments/assets/43134384-a9b5-4b23-b40a-e4b9b1562337)

# **Environment** 

Rocket.chat version: 4.56.0
Device: Samsung S24 ultra 
One UI version: 6.1
Android version: 14

**Comment by reetp** [comment]
Please fill out a bug template properly.

We need your server details.

This may help.

https://gist.github.com/reetp/a66149d5f060f260643a353ca7067a98#information-that-support-or-devs-will-require

Full server info + proxy details please.

Next, this is a screnshot of 4.56 with one of my servers. No issues.

![Screenshot_20241230_005232_RocketChat.jpg](https://github.com/user-attachments/assets/7a232f59-3b52-43af-a3df-a268b45707da)

You should test on open.rocket.chat

> However, the web version takes an excessively long time to load, sometimes 2-3 minutes on both desktop and mobile devices, making this workaround impractical

See the above gist for "slow connections".

This is likely an issue with your deployment and not a bug.

**Comment by MRJOHN5ON** [comment]
After looking into this further, I realized I misunderstood how Rocket.Chat works. I now understand that functionality and performance can vary depending on how the server is deployed and configured by the organization using it. The issues I reported seem to be specific to my employer's deployment rather than a problem with Rocket.Chat itself.

I apologize for any confusion or for taking up your time unnecessarily. I’ll follow up internally to address these deployment-specific issues. Thank you for your prompt response
